packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -753,15 +756,22 @@ const styles = StyleSheet.create({
         border: "none",
         outline: "none",
         padding: 0,
-        minWidth: spacing.xxxSmall_4,
-        width: "auto",
-        display: "inline-grid",
-        gridArea: "1 / 2",
         ":focus-visible": {
             outline: "none",
             border: "none",
+            boxShadow: "none",
+        },
+        [":active:not([aria-disabled='true']):not([readonly])" as any]: {
```

Comment:
Needs to have high specificity to override the press styling in TextField 🥲 The press styling is like this in TextField so that it is not shown when the field is disabled or in a readonly state